File: backend/models/roles.py
"""
Roles model.
"""
from typing import Optional
from pydantic import BaseModel

class Roles(BaseModel):
    """Roles model."""
    role_id: str    user_id: str    application_id: str    role_name: str    role_type: str    permissions: str    created_at: int    updated_at: int    active: str
    class Config:
        """Model configuration."""
        from_attributes = True

File: frontend/src/models/roles.ts
/**
 * roles model.
 */
export interface Roles {
  role_id: string;
  user_id: string;
  application_id: string;
  role_name: string;
  role_type: string;
  permissions: string;
  created_at: number;
  updated_at: number;
  active: string;
}